.circleci/config.yml

```diff
@@ -1,11 +1,42 @@
-version: 2
+version: 2.1
+
+commands:
```

Comment:
I think there was no cache for the sake of simplicity. And you will not avoid token problem anyway.

Reply:
Except that I had multiple build failure because of it.